packages/flutter/test/material/dropdown_test.dart

```diff
@@ -3165,4 +3166,77 @@ void main() {
         ..rrect(rrect: const RRect.fromLTRBXY(0.0, 0.0, 112.0, 47.0, 2.0, 2.0), color: Colors.grey[50], hasMaskFilter: false)
     );
   });
+
+  group('feedback', () {
+    late FeedbackTester feedback;
```

Comment:
`late final`?

Reply:
Actually, if I change it to final the tests succeeding the test 'Dropdown with enabled feedback' would fail with the error `LateInitializationError: Local 'feedback' has already been initialized.` which is expected since `setUp()` would be called again in those tests and since `FeedbackTester` is final, it would error out.

Reply:
Sorry, I don't use test groups; didn't realize that setUp()/tearDown() was per test not per group.